integration/fuse/src/main/java/alluxio/fuse/AlluxioJniFuseFileSystem.java

```diff
@@ -668,6 +689,77 @@ public String getFileSystemName() {
     return mFsName;
   }
 
+  @Override
+  public void umount() {
+    // Release operation is async, we will try our best efforts to
+    // close all opened file in/out stream before umounting the fuse
+    if (!mCreateFileEntries.isEmpty() || !mOpenFileEntries.isEmpty()) {
+      LOG.info("Waiting for all file out stream closed");
```

Comment:
Can we add something to this message, like the path we are umounting?

Reply:
Yeah, added